subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -370,490 +370,64 @@ Once a configuration is resolved it is immutable. Changing its state or the stat
 
-.Repository types
-[cols="a,a", options="header"]
-|===
-| Type
-| Description
-
-| <<sub:maven_central,Maven central repository>>
-| A pre-configured repository that looks for dependencies in Maven Central.
-
-| <<sub:maven_jcenter,Maven JCenter repository>>
-| A pre-configured repository that looks for dependencies in Bintray's JCenter.
-
-| <<sub:maven_google,Maven Google repository>>
-| A pre-configured repository that looks for dependencies in Google's Maven repository.
-
-| <<sub:maven_local,Maven local repository>>
-| A pre-configured repository that looks for dependencies in the local Maven repository.
-
-| <<sub:maven_repo,Maven repository>>
-| A Maven repository. Can be located on the local filesystem or at some remote location.
-
-| <<sec:ivy_repositories,Ivy repository>>
-| An Ivy repository. Can be located on the local filesystem or at some remote location.
-
-| <<sec:flat_dir_resolver,Flat directory repository>>
-| A simple repository on the local filesystem. Does not support any meta-data formats.
-|===
-
-
-[[sub:maven_central]]
-==== Maven central repository
-
-To add the central Maven 2 repository (https://repo.maven.apache.org/maven2[]) simply add this to your build script:
-
-++++
-<sample id="mavenCentral" dir="userguide/artifacts/defineRepository" title="Adding central Maven repository">
-                <sourcefile file="build.gradle" snippet="maven-central"/>
-            </sample>
-++++
-
-Now Gradle will look for your dependencies in this repository.
-
-[[sub:maven_jcenter]]
-==== Maven JCenter repository
-
-http://bintray.com[Bintray]'s JCenter is an up-to-date collection of all popular Maven OSS artifacts, including artifacts published directly to Bintray.
-
-To add the JCenter Maven repository (https://jcenter.bintray.com[]) simply add this to your build script:
-
-++++
-<sample id="mavenJcenter" dir="userguide/artifacts/defineRepository" title="Adding Bintray's JCenter Maven repository">
-                <sourcefile file="build.gradle" snippet="maven-jcenter"/>
-            </sample>
-++++
-
-Now Gradle will look for your dependencies in the JCenter repository. _jcenter()_ uses HTTPS to connect to the repository. If you want to use HTTP you can configure `jcenter()`:
-
-++++
-<sample id="mavenJcenter" dir="userguide/artifacts/defineRepository" title="Using Bintrays's JCenter with HTTP">
-                <sourcefile file="build.gradle" snippet="maven-jcenter-http"/>
-            </sample>
-++++
-
-[[sub:maven_google]]
-==== Maven Google repository
-
-The Google repository hosts Android-specific artifacts including the Android SDK. For usage examples please the [relevant documentation](https://developer.android.com/studio/build/dependencies.html#google-maven).
++++++
```

Comment:
Totally optional nitpick: this could be a bit wider.

Reply:
done